documentation/markdown/accesories/modular_DIN_rail_case/datasheet/datasheet.md

```diff
@@ -53,29 +54,29 @@ Standard ring for balenaFin, exposes the following ports:
 | HAT | IDC cable |
 | ![Fin mapping top](pictures/hat.jpg) | ![Fin mapping bottom](pictures/idc_cable.jpg) |
 
-The terminal block ring extends balenaFin I/O headers to the exterior of the case. Comes with a HAT and IDC cable. 
+The terminal block ring extends balenaFin I/O headers to the exterior of the case. Comes with a HAT and IDC cable.  
 
 <div class="page-break"></div>
 
 ### 1.2.1 Terminal block ring pin map
 
 | **Fin header pin #** | **Co-processor header pin #** | **Case terminal pin #** | **Pin name** |
 | --- | --- | --- | --- |
-| 2  | NC | 1  | 5V |
-| 6  | NC | 2  | BCM10 (MOSI) |
-| 8  | NC | 3  | GND | 
-| 10 | NC | 4  | BCM9 (MISO) |
-| 3  | NC | 5  | BCM14 (TXD) |
-| 5  | NC | 6  | BCM11 (SCLK) |
-| 12 | NC | 7  | BCM15 (RXD) |
-| 16 | NC | 8  | PD14 |
-| 19 | NC | 9  | BCM2 (SDA) |
-| 21 | NC | 10 | PA2 |
-| 23 | NC | 11 | BCM3 (SCL) |
-| NC | 1  | 12 | PA3 |
-| NC | 3  | 13 | BCM18 (PWM0) |
-| NC | 5  | 14 | PF6 |
-| NC | 13 | 15 | BCM23 |
+| 2  | NC | 1  | 5V             |
+| 19 | NC | 2  | BCM10 (MOSI)   |
+| 6  | NC | 3  | GND            |
+| 21 | NC | 4  | BCM9 (MISO)    |
+| 8  | NC | 5  | BCM14 (TXD)    |
+| 23 | NC | 6  | BCM11 (SCLK)   |
+| 10 | NC | 7  | BCM15 (RXD)    |
+| NC | 1  | 8  | PD14           |
+| 3  | NC | 9  | BCM2 (SDA)     |
+| NC | 3  | 10 | PA2            |
+| 5  | NC | 11 | BCM3 (SCL)     |
+| NC | 5  | 12 | PA3            |
+| 12 | NC | 13 | BCM18 (PWM0)   |
+| NC | 13 | 14 | PF6            |
+| 16 | NC | 15 | BCM23          |
```

Comment:
@ntzovanis is it worth adding that pin 16 (on the case terminal) is NC?

Reply:
@Bucknalla  The header on the PCB is not mentioned on the datasheet, only the phoenix connector on the case (15 pins). It might be more confusing if we add the PCB header to the table.